pom.xml

```diff
@@ -14,57 +14,45 @@
     </properties>
 
     <build>
-        <pluginManagement>
```

Comment:
Please fix `[WARNING] File encoding has not been set, using platform encoding UTF-8, i.e. build is platform dependent!`